cmake/Modules_CUDA_fix/upstream/FindCUDA/select_compute_arch.cmake

```diff
@@ -105,8 +105,11 @@ endif()
 
 if(NOT CUDA_VERSION VERSION_LESS "11.8")
   list(APPEND CUDA_KNOWN_GPU_ARCHITECTURES "Ada")
+  list(APPEND CUDA_KNOWN_GPU_ARCHITECTURES "Hopper")
   list(APPEND CUDA_COMMON_GPU_ARCHITECTURES "8.9")
+  list(APPEND CUDA_COMMON_GPU_ARCHITECTURES "9.0" "9.0+PTX")
```

Comment:
`"9.0+PTX"` should not be appended here. We should update line 116 instead.

Also, `"8.6+PTX"` should not be appended in line 97. We should add 
```
list(APPEND CUDA_COMMON_GPU_ARCHITECTURES "8.6+PTX")
```
in line 103.